tests/js/navIconsTests.js

```diff
@@ -0,0 +1,67 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+    "use strict";
+
+    fluid.registerNamespace("gpii.tests");
+
+    gpii.tests.hasClass = function (elementName, element, selector, expected) {
+        jqUnit.assertEquals(elementName + " does " + (expected ? "" : "not") + " have " + selector + " applied", expected, element.hasClass(selector));
```

Comment:
This message is also fairly complex in its structure. Is there a simpler way to convey this?